packages/react/ds/src/accordion/accordion.tsx

```diff
@@ -0,0 +1,7 @@
+type Props = {
+  children: React.ReactNode;
+};
+
+export const Accordion = ({ children }: Props) => {
+  return <div className="gi-max-w-[700px]">{children}</div>;
```

Comment:
Don't use this hardcoded size.
If you want to define a max width, try to evaluate `max-w-prose`

Reply:
This was intended to be used as a temporary measure because as agreed previously, we will add the styles ( including a max width ) at a later iteration. But sure I can use prose for now